README.md

```diff
@@ -208,52 +209,102 @@ ROS_NAMESPACE=zivid_camera rosrun zivid_camera zivid_camera_node _frame_id:=zivi
 
 ## Services
 
-`capture` ([zivid_camera/Capture](./zivid_camera/srv/Capture.srv))
-> Invoke this service to trigger a 3D capture. See section [Configuration](#configuration) for how to
-> configure the 3D capture settings. The resulting point cloud is published on topic `points`, color image
-> is published on topic `color/image_color`, and depth image is published on topic `depth/image_raw`.
-> Camera calibration is published on topics `color/camera_info` and `depth/camera_info`.
+### capture_assistant/suggest_settings
```

Comment:
How exactly would I launch this service from the command line? It's not obvious to me how to specify the two arguments.

Should we add an example-line here to demonstrate?

Reply:
We could add examples of how to launch from command line. Command line is more of a troubleshooting/debugging tool. I dont think it should be added here as it is noise. Maybe under FAQ. 

You can try rosservice call /zivid_camera/capture_assistant/suggest_settings <tab><tab>